src/components/App.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import { BrowserRouter } from "react-router-dom";
+
+import AppRoutes from "./routes/AppRoutes";
+
+const App = () => (
+  <BrowserRouter>
+    <Helmet>
```

Comment:
if unit testing is in your tasks list, probably this browser router should be moved to your index.tsx file, it would make your life easier